department-of-veterans-affairs/caseflow
# Queue Attorney check out | Delete an issue

Resolves #4478 

### Description
Adds the Delete Issue flow

### Acceptance Criteria 
- [ ] Verify that when a user clicks delete issue from the "Edit Issue" page a modal with the title 'Delete Issue" appears
- [ ] Verify that the modal has the following copy: "You are about to permanently delete this issue. To delete please click the "Delete issue" button or click the "Go back link"  to return the previous screen. "
- [ ] Verify that the modal has a link text on the left with the following copy "Close" 
- [ ] Verify that there is a red secondary button labeled "Delete issue" 
- [ ] Verify that the Deleted issues no longer exist in VACOLS when the user confirms the delete within the modal

### Testing Plan
1. Go to the Draft Decision checkout flow, Edit Issue.
2. Click Delete issue, verify modal appears.
3. Click Delete issue, verify you're taken back to `/queue/tasks/#{appeal_id}/dispositions`, minus 1 issue, with delete success banner visible.
